hasher-matcher-actioner/hmalib/common/message_models.py

```diff
@@ -89,25 +95,76 @@ def from_match_message_action_label_and_action_rules(
 
 
 @dataclass
-class WritebackMessage(MatchMessage):
+class WritebackMessageConfig:
     """
+    Simple holder for getting typed environment variables
+    """
+
+    writebacks_queue_url: str
+    sqs_client: SQSClient
+
+    @classmethod
+    @lru_cache(maxsize=None)
+    def get(cls):
+        HMAConfig.initialize(os.environ["CONFIG_TABLE_NAME"])
```

Comment:
(repeated note): env values used in the API lambda but not declared in the top level module are hard to debug and easy to miss. We've gotten around this before by having this init occur within api_root.py. Thoughts?

Reply:
I follow your comment here. Will move to `matches.get_matches_api` to mirror `datasets_api`